scripts/manual_add_to_db.py

```diff
@@ -77,6 +77,17 @@
             "video_links": [],
             "new": True
         },
+        {
+            "_id" : "vector-rake",
+            "title": "Vector Rake",
+            "description": "Updates docs with a subset of keywords selected using Rake and Cosine distance in vector space",
```

Comment:
I think a higher level description is needed here for the end user. 
Something like: 
"Keyphrase extraction using a nearest-neighbor algorithm on top of the normal RAKE algorithm"